deepchecks/nlp/utils/text_properties.py

```diff
@@ -10,14 +10,56 @@
 #
 """Module containing the text properties for the NLP module."""
 import string
+import time
 import warnings
-from typing import Dict, List, Sequence
+from typing import Dict, List, Optional, Sequence
 
 import numpy as np
 
+from deepchecks.utils.function import run_available_kwargs
+
 __all__ = ['calculate_default_properties']
 
 
+def property_import_error(property_name: str, package_name: str) -> ImportError:
+    """Raise an ImportError for a property that requires a package."""
+    return ImportError(
+        f'property {property_name} requires the {package_name} python package. '
+        f'To get it, run "pip install {package_name}".')
+
+
+def get_transformer_model(property_name: str, model_name: str, device: Optional[str] = None):
+    """Get the transformer model and decide if to use optimum.onnxruntime."""
+    if device is None or device == 'cpu':
+        try:
+            from optimum.onnxruntime import ORTModelForSequenceClassification  # pylint: disable=import-outside-toplevel
+        except ImportError as e:
+            error_message = f'The device was set to {device} while computing the {property_name} property, in which' \
+                            f' case deepchecks resorts to accelerating the inference using optimum. Either set the ' \
+                            f'device according to your hardware, or install the required libraries by running:\n' \
+                            f'pip install optimum onnx onnxruntime'
```

Comment:
I think we should have some solution for people who want to use the nlp properties and not get so many errors when they start using, which would be extremely annoying. Not sure if this is easy or not, but like having "pip install deepchecks[nlp-properties] or something. WDYT?

Reply:
done